manifests/init.pp

```diff
@@ -20,6 +20,7 @@
   Integer[0, 65535]                             $port                            = 53,
   Array[String[1]]                              $interface                       = [],
   Boolean                                       $interface_automatic             = false,
+  Optional[String[1]]                           $interface_automatic_ports       = undef,
```

Comment:
Please add some words about this new parameter above:

```
@param interface_automatic_ports
  Short description of this parameter.
```

Reply:
i had manually updated the REFERENCE.md file but done it this way now and run the rake task